src/routes/text/+page.svelte

```diff
@@ -503,6 +522,13 @@
             </div>
         </div>
     {/if}
+    {#if scrollingUp && bottomNavBarEnabled && !$selectedVerses.length > 0}
+        <div class="bottom-nav-bar" transition:slide={{ duration: 300 }}>
+            <div>
```

Comment:
The Empty divs not necessary, but of course the one with the transition is.  Is the `bottom-nav-bar` class  required?

Reply:
Not really.  I put it in originally because I thought I would have some css that would go with it and then left it to make it easier to spot.  Removed.